web/admin/application/configs/klear/BrandsList.yaml

```diff
@@ -20,11 +20,11 @@ production:
           name: true
           invoice.nif: true
           logo: true
-          defaultTimezoneId: true
-          languageId: true
+          defaultTimezone: true
+          language: true
           domain_users: true
-          defaultTimezoneId: true
-          languageId: true
+          defaultTimezone: true
+          language: true
```

Comment:
Not part of the PR changes but `defaultTimezone`and `language` are present twice in this list